src/cli/cli_dataset.cpp

```diff
@@ -205,12 +205,17 @@ template <> otError Dataset::Process<Cmd("activetimestamp")>(Arg aArgs[])
     {
         if (sDataset.mComponents.mIsActiveTimestampPresent)
         {
-            OutputLine("%lu", sDataset.mActiveTimestamp);
+            OutputLine("%lu", sDataset.mActiveTimestamp.mSeconds);
         }
     }
     else
     {
-        SuccessOrExit(error = aArgs[0].ParseAsUint64(sDataset.mActiveTimestamp));
+        uint64_t activeTimestampSeconds;
+
+        SuccessOrExit(error = aArgs[0].ParseAsUint64(activeTimestampSeconds));
```

Comment:
may allow configuring the whole timestamp ?

e.g. `<seconds>.<ticks>.A`

Reply:
Trying to keep the CLI interface as it is, because there are applications which depends on the CLI format. This also avoids unnecessary changes in this PR.